ERROR - 2022-08-08 07:22:37 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1380): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1411): CodeIgniter\Database\BaseConnection->listTables()
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(167): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#15 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1380): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1411): CodeIgniter\Database\BaseConnection->listTables()
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(167): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#14 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#15 {main}
ERROR - 2022-08-08 07:23:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1380): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1411): CodeIgniter\Database\BaseConnection->listTables()
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(167): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#15 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1380): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1411): CodeIgniter\Database\BaseConnection->listTables()
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(167): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#14 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#15 {main}
ERROR - 2022-08-08 07:23:48 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1380): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1411): CodeIgniter\Database\BaseConnection->listTables()
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(167): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#15 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#16 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1380): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1411): CodeIgniter\Database\BaseConnection->listTables()
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(167): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#14 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#15 {main}
ERROR - 2022-08-08 07:47:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('localhost', '', '', 'ci4', 3306, '', 0)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Config.php(105): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Seeder.php(96): CodeIgniter\Database\Config::forge(NULL)
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Seed->run(Array)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#13 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Config.php(105): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Seeder.php(96): CodeIgniter\Database\Config::forge(NULL)
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Seed->run(Array)
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#12 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#13 {main}
CRITICAL - 2022-08-08 07:47:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/Config.php(105): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH/Database/Seeder.php(96): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH/Commands/Database/Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH/CLI/Commands.php(63): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 SYSTEMPATH/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index([...])
 8 SYSTEMPATH/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', [...])
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
10 SYSTEMPATH/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
12 ROOTPATH/spark(98): CodeIgniter\CLI\Console->run()
ERROR - 2022-08-08 07:47:28 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('localhost', '', '', 'ci4', 3306, '', 0)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Config.php(105): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Seeder.php(96): CodeIgniter\Database\Config::forge(NULL)
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Seed->run(Array)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('db:seed', Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#13 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Config.php(105): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
#3 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Database/Seeder.php(96): CodeIgniter\Database\Config::forge(NULL)
#4 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/Commands/Database/Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
#5 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Seed->run(Array)
#6 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('db:seed', Array)
#7 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#8 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#9 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#10 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 /Users/jeanchristophe/App/apiphp/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#12 /Users/jeanchristophe/App/apiphp/spark(98): CodeIgniter\CLI\Console->run()
#13 {main}
CRITICAL - 2022-08-08 07:47:28 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/Database.php(67): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/Config.php(105): CodeIgniter\Database\Database->loadForge(Object(CodeIgniter\Database\MySQLi\Connection))
 3 SYSTEMPATH/Database/Seeder.php(96): CodeIgniter\Database\Config::forge(null)
 4 SYSTEMPATH/Commands/Database/Seed.php(69): CodeIgniter\Database\Seeder->__construct(Object(Config\Database))
 5 SYSTEMPATH/CLI/Commands.php(63): CodeIgniter\Commands\Database\Seed->run([...])
 6 SYSTEMPATH/CLI/CommandRunner.php(65): CodeIgniter\CLI\Commands->run('db:seed', [...])
 7 SYSTEMPATH/CLI/CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index([...])
 8 SYSTEMPATH/CodeIgniter.php(891): CodeIgniter\CLI\CommandRunner->_remap('index', [...])
 9 SYSTEMPATH/CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
10 SYSTEMPATH/CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
12 ROOTPATH/spark(98): CodeIgniter\CLI\Console->run()
